src/libsystemd/sd-journal/journal-def.h

```diff
@@ -57,7 +57,6 @@ struct ObjectHeader {
         uint8_t flags;
         uint8_t reserved[6];
         le64_t size;
-        uint8_t payload[];
```

Comment:
The preceding fields have size that is a multiple of 8 bytes, so the end of `.size` is aligned, so offsetof(ObjectHeader, payload) == sizeof(ObjectHeader)`, good.